components/browse-decks/DeckBox.js

```diff
@@ -46,6 +46,15 @@ const DeckBox = styled.a`
   :disabled {
     cursor: not-allowed;
   }
+
+  :focus {
+    outline: none;
+    -webkit-box-shadow: 0px 0px 10px 2px
+      ${({ theme }) => theme.colors.primaryMuted};
+    -moz-box-shadow: 0px 0px 10px 2px
```

Comment:
I don't think we should add flags. Most browsers have stopped relying on them, and we risk having different styles for different browsers if we forget to update one of these.